starlark/experimental/go.star

```diff
@@ -0,0 +1,128 @@
+# vi:syntax=python
+
+load("github.com/mesosphere/dispatch-catalog/starlark/stable/pipeline@master", "imageResource", "storageResource", "resourceVar")
+load("github.com/mesosphere/dispatch-catalog/starlark/experimental/buildkit@master", "buildkitContainer")
+
+__doc__ = """
+# Go
+
+Provides methods for building and testing Go modules.
+
+Import URL: `github.com/mesosphere/dispatch-catalog/starlark/stable/go`
+"""
+
+def go_test(git, name, paths=None, image="golang:1.13.0-buster", inputs=None, **kwargs):
+    """
+    Run Go tests and generate a coverage report.
+    """
+
+    if not paths:
```

Comment:
I usually see `if paths is None` instead of `if not paths`, not sure which is more common.

Reply:
(nit alert)